transmogrifier/config.py

```diff
@@ -173,7 +174,8 @@ def load_external_config(
     with open(file_path, "rb") as config_file:
         if file_type == "json":
             return json.load(config_file)
-        elif file_type == "xml":
+        elif file_type == "xml":  # noqa: RET505
             return BeautifulSoup(config_file, "xml")
         else:
-            raise ValueError("Unrecognized file_type parameter: %s", file_type)
+            message = f"Unrecognized file_type parameter: {file_type}"
+            raise ValueError(message)
```

Comment:
I think this was a good call too.  I looked at the code as-is, and I agree this is the most readable form.  Sure, we might get away with removing an `elif` or `else` here and there, but as-is it reads very clearly.